hugr-core/src/std_extensions.rs

```diff
@@ -21,8 +21,10 @@ pub fn std_reg() -> ExtensionRegistry {
         collections::EXTENSION.to_owned(),
         logic::EXTENSION.to_owned(),
         ptr::EXTENSION.to_owned(),
-    ])
-    .unwrap()
+    ]);
+    reg.validate()
```

Comment:
Many examples here and elsewhere, I wonder if it's worth adding to ExtensionRegistry
```
   fn validated(mut self) -> Result<Self, ....> {
      self.validate()?;
      self
   }
```

Reply:
We'll call `ExtRegistry::validate` on each `Hugr::validate` as soon as #1738 gets merged, so this shouldn't be needed.